website/docs/r/elasticsearch_domain.html.markdown

```diff
@@ -109,6 +109,7 @@ The following attributes are exported:
 * `arn` - Amazon Resource Name (ARN) of the domain.
 * `domain_id` - Unique identifier for the domain.
 * `endpoint` - Domain-specific endpoint used to submit index, search, and data upload requests.
+* `kibana_endpoint` - Domain-specific endpoint for kibana without https scheme, the same as `endpoint`.
```

Comment:
By "the same as `endpoint`", do you mean the same as in "without https"?
If so, I think this is misleading with the beginning of the sentence, what do you think? :)

Reply:
Exactly. Now removed those confusing words.